lark/exceptions.pyi

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+from typing import Dict, Iterable, Callable
+from .tree import Tree
+
+
+class LarkError(Exception):
+    pass
+
+
+class GrammarError(LarkError):
+    pass
+
+
+class ParseError(LarkError):
+    pass
+
+
+class LexError(LarkError):
+    pass
+
+
+class UnexpectedInput(LarkError):
+    pos_in_stream: int
+
+    def get_context(self, text: str, span: int = 40):
```

Comment:
Isn't it a bad idea to write the default here? It may change in the future.

Or is it unused?

Reply:
Default value can show as `span: int = ...` if it might changed.
Or just hide it if it's a developer option.

Reply:
Can't mypy take it from the actual function? Or is there a way to use a CONST from the code (i.e. `DEFAULT_CONTEXT_SPAN`, so there will be one source of truth?

Reply:
The actual value is not important in static type checking.
It at most makes the IDE display its preset value.

Reply:
My point is that if a value will be displayed, it has to be the correct one, and change if the code changes. Otherwise, it's better to not have a value.

Reply:
I think the default value is just to remind the user that this is an optional parameter of default behavior.
Regardless the implementation, they should keep refer to docstring or documentation if they are not truly understand the function.

Reply:
A quick search revealed the correct way to write it:

    for arguments with a default, use ... instead of the actual default;

https://github.com/python/typeshed/blob/master/CONTRIBUTING.md#conventions